Dockerfile

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile1
+ARG VERSION=20.04
+FROM ubuntu:$VERSION
```

Comment:
You could also use `python:3.7`, which is what I've been using for the web app, which will preclude some of the apt-install steps.  Wouldn't be surprised if this has some disadvantages though.